mautrix_twitter/db/reaction.py

```diff
@@ -51,12 +52,15 @@ async def insert(self) -> None:
             self.tw_receiver,
             self.tw_sender,
             self.reaction.value,
+            self.tw_reaction_id,
         )
 
-    async def edit(self, mx_room: RoomID, mxid: EventID, reaction: ReactionKey) -> None:
+    async def edit(
+        self, mx_room: RoomID, mxid: EventID, reaction: ReactionKey, tw_reaction_id: int | None
+    ) -> None:
         q = (
             "UPDATE reaction SET mxid=$1, mx_room=$2, reaction=$3 "
-            "WHERE tw_msgid=$4 AND tw_receiver=$5 AND tw_sender=$6"
+            "WHERE tw_msgid=$4 AND tw_receiver=$5 AND tw_sender=$6 AND tw_reaction_id=$7"
```

Comment:
This looks like it's the wrong way around (it should store the new reaction ID)

Reply:
You're totally right - even my editor was showing me I had the wrong one ;)